Source/Moq.Analyzers.Test/PackageTests.Baseline#contents.verified.txt

```diff
@@ -0,0 +1,12 @@
+﻿/
+|-- Moq.Analyzers.nuspec
+|-- analyzers
+|   |-- dotnet
+|   |   |-- cs
+|   |   |   |-- Moq.Analyzers.dll
+|-- lib
```

Comment:
Since this is an analyzer package and there's no "shipping" code, I don't think the `lib/` directory should be in the package. Tracked by #31.

Reply:
Related to #8?

Reply:
Yes, likely. `0.0.8` puts it in both places like this change has. `0.0.9` is missing the analyzer.

So I think #8 will be fixed in the next release either way, and also we should ship the analyzer as a lib.